site/docs/v1/tech/apis/api-keys.adoc

```diff
@@ -0,0 +1,172 @@
+---
+layout: doc
+title: API Key APIs
+description: APIs for creating, retrieving, updating and deleting API Keys
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.26.0
+====
+
+The FusionAuth APIs are primarily secured using API keys. This API can only be accessed using an API key that is set as a keyManager. In order to retrieve, update or delete an API key, an API key with equal or greater permissions must be used. A "tenant-scoped" API key can retrieve, create, update or delete an API key for the same tenant. This page describes APIs that are used to manage API keys.
+
```

Comment:
I would add a section (or sections) here that covers the following:

* how do I create a keymanager API key (including a picture of the UI)
* a list of possible HTTP methods (get, put, delete, post, patch)
* a list of API endpoints that can be set
* an explanation that if no api endpoints are set, that makes it a super key with privileges for all endpoints.

Reply:
Can we link to this page instead? https://fusionauth.io/docs/v1/tech/apis/authentication/

Reply:
That makes total sense! Sorry, forgot about that page.